### PR TITLE
Positions on grid for demo

### PR DESCRIPTION
**Pull request by lucieho**
This PR adds a test class that uploads fake users with their fake positions to the Firestore Livegrid.
This is for demo purposes. 
Fake users are all located at EPFL, either infected either not.
1 area with a lot of users, another with less.

### CODE REVIEW

**Review by AdrienRem — COMMENTED** [review]
HI ! I don't understand why you changed the FirebaseOfflineCacheTest file. Could you clarify ?

**Comment by lucieho** [comment]
> HI ! I don't understand why you changed the FirebaseOfflineCacheTest file. Could you clarify ?
HI ! 
I didn't understand why uploads from DataForDemo were not appearing on Firestore after several attempts. I decided to test, in FirebaseOfflineCacheTest, that the data was retrieved/written from/to the Firestore backend server and not from/to the local cache. 
(I had to add a few lines of code in synchronizeLocalChangeWithBackendWhenBackOnline() for this)

**Review by AdrienRem — APPROVED** [review]
Okay, Thanks for your explanation !

File: app/src/androidTest/java/ch/epfl/sdp/DataForDemo.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    private static double DENSE_INITIAL_EPFL_LATITUDE = 46.51700;
+    private static double DENSE_INITIAL_EPFL_LONGITUDE = 6.56600;
+    private static double SPARSE_INITIAL_EPFL_LATITUDE = 46.51800;
+    private static double SPARSE_INITIAL_EPFL_LONGITUDE = 6.56700;
+
+    /**
+     * Generate 30 users around 46.51700,6.56600 and 5 users around 46.51800, 6.56700.
+     * These latitude, longitude correspond to areas at EPFL.
+     */
+     @Test
```
[the remark this answers is not in the document]
Comment (TheBlueHawk) [reply app/src/androidTest/java/ch/epfl/sdp/DataForDemo.java:44]: First part (dense location) looks good to me, second one can be improved by extracting a method for the upload of the couple carrier-location

[the remark this answers is not in the document]
Comment (TheBlueHawk) [reply app/src/androidTest/java/ch/epfl/sdp/DataForDemo.java:44]: This can be reduced using the suggestion above

**Review by TheBlueHawk — CHANGES_REQUESTED** [review]
Apart the little refactoring needed, good job, this will be incredibly useful for next demos!

**Comment by lucieho** [comment]
Thanks, extract method done.

**Review by TheBlueHawk — APPROVED** [review]

**Pull request merged**